bsb/connectivity/point_cloud/cloud_cloud_intersection.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
+from bsb.connectivity import ConnectionStrategy
+from bsb import config
+from bsb.connectivity.point_cloud.geometric_shapes import ShapesComposition
+
+
+@config.node
+class CloudToCloudIntersection(ConnectionStrategy):
```

Comment:
```suggestion


@config.node
class CloudHemitype(Hemitype):
  cloud_names = config.list(type=str)


@config.node
class CloudToCloudIntersection(ConnectionStrategy):
  presynaptic = config.attr(type=CloudHemitype)
  postsynaptic = config.attr(type=CloudHemitype)
```

Without this extension of the hemitype node, you will get `unknown attribute 'cloud_names'` warnings.